pkg/controllers/scalablenodegroup/v1alpha1/suite_test.go

```diff
@@ -53,24 +54,75 @@ var _ = AfterSuite(func() {
 
 var _ = Describe("Examples", func() {
 	var ns *environment.Namespace
+	var desiredReplicas = ptr.Int32(5)
+	var dummyMessage = "test message"
 	var sng *v1alpha1.ScalableNodeGroup
 
 	BeforeEach(func() {
 		var err error
 		ns, err = env.NewNamespace()
 		Expect(err).NotTo(HaveOccurred())
 		sng = &v1alpha1.ScalableNodeGroup{}
+		fakeCloudProvider.NodeGroupStable = true
+		fakeCloudProvider.NodeGroupMessage = ""
+		fakeCloudProvider.WantErr = nil
+		sng.Spec.Replicas = desiredReplicas
+		sng.Spec.ID = "dummyID"
```

Comment:
I don't understand why this is necessary.

Reply:
Yeah, I can probably get away with it.